### ISSUE TITLE
Do not bundle babel-polyfill with library. Let it be an external peer dependency

### ISSUE THREAD

**Issue by anandanand84** [transcranial/keras-js#109]
Shipping babel polyfill with the library doesnt play nice with other libraries.

https://github.com/transcranial/keras-js/blob/master/src/index.js#L1

 If an application depends on multiple libraries and everything ships with babel-polyfill it will cause
```
Uncaught Error: only one instance of @babel/polyfill is allowed
```

Even in cases where the other application doesnt ship with babel polyfill but depend on babel-polyfill, it will fail since the the babel polyfill will be loaded later when this(keras-js) dependency is loaded. 

Only solution is to load the babel polyfill first from a cdn or the end application add it as a dependency in their build process

**Comment by levithomason** [comment]
Agreed.  Shipping the polyfill causes unwarranted side-effects for apps loading keras-js as well.  It would be ideal if keras-js provided pre-compiled targets.  This way users could opt-in to loading the package that suited their needs.

**Comment by tiansivive** [comment]
Please fix this
